.circleci/cluster/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM cfy_manager_image
+
+COPY *.rpm /opt
```

Comment:
why not `/tmp`?

Reply:
the tmp path (`/tmp/workspace/x86_64`) is a detail of the circleci side so i think i prefer that all references to it stay in the circleci config file, not here
additionally `COPY` only supports copying from the dockerfile dir, so this would have to become `ADD` and eh, i prefer `COPY` because `ADD` does waaay to many things

Reply:
oh you mean why not copy TO `/tmp`? i don't think that will be preserved between the `docker build` and the actual run

Reply:
oh right, but it doesn't need to be. OK, right